css-color/Overview.bs

```diff
@@ -1872,18 +1890,8 @@ Predefined colorspaces: ''srgb'', ''dci-p3'', and ''rec2020''.</h3>
 				<tr><th>Green chromaticity</th><td>x=0.170 y=0.797</td></tr>
 				<tr><th>Blue chromaticity</th><td>x=0.131 y=0.046</td></tr>
 				<tr><th>White chromaticity</th><td>x=0.3127 y=0.3290 (D65)</td></tr>
-				<tr><th>Transfer function</th><td>see below</td></tr>
+				<tr><th>Transfer function</th><td>1/2.4</td></tr>
```

Comment:
I agree that the normative reference should be Rec.2020. The "see below" transfer function with the high precision coefficients was copied from that specification. Just saying that is is aboyt 1/2.4 is just as wrong as saying that sRGB is "about 1/2.2".